src/libexpr/eval-settings.hh

```diff
@@ -127,6 +127,9 @@ struct EvalSettings : Config
 
     Setting<unsigned int> maxCallDepth{this, 10000, "max-call-depth",
         "The maximum function call depth to allow before erroring."};
+
+    Setting<bool> builtinsTraceDebugger{this, false, "builtins-trace-debugger",
+        "Whether to enter the debugger on `builtins.trace` calls."};
```

Comment:
Please add links to documentation of `debugger` and `builtins.trace`. People who start reading from the command reference otherwise won't know what either means or where to find that information.

Reply:
Done.

<img width="732" alt="image" src="https://github.com/NixOS/nix/assets/15312184/e6d57457-d163-4ce2-bf22-50cfafcdd00c">
<img width="782" alt="image" src="https://github.com/NixOS/nix/assets/15312184/e66e4862-aa4e-43a6-afde-f58177c9a223">